platform/src/main/resources/config/anchor-config-default-values.yaml

```diff
@@ -330,62 +330,75 @@ metrics:
 #########################
 ## The events being sent from the platform are the ones described in the `Events Schema.yml` file
 events:
-
   ## @param: enabled
   ## @type: bool
   ## Whether to enable events.
   #
   enabled: false
 
+  ## @param: event_type_to_queue
+  ## @type: map
+  ## Mapping of the event type to the queue name that messages are published to
+  event_type_to_queue:
+    quote_created: ap_event_quote_created
+    transaction_created: ap_event_transaction_created
+    transaction_status_changed: ap_event_transaction_status_changed
+    transaction_error: ap_event_transaction_error
```

Comment:
We used to have the option `all: ap_event_single_queue` as well. Did you intend to remove that? I don't think that's needed TBH.